usr/share/pipewire/pipewire-rnnoise.conf

```diff
@@ -0,0 +1,50 @@
+context.properties = {
```

Comment:
This file does not get loaded by PipeWire by default. I had to create a service file for it.

I went back and tested a variety of different configuration locations and none of them worked. The only thing that did work was my service to load the configuration by running `pipewire -c`.

- /etc/pipewire/pipewire.conf.d/ = `pipewire` user service crashes with no output
- /usr/share/pipewire/ = Nothing
- /usr/share/pipewire/client.conf.d/ = Nothing
- /usr/share/pipewire/pipewire.conf.d/ = `pipewire` user service crashes with no output
- /usr/share/pipewire/pipewire-pulse.conf.d/ = `pipewire-pulse` user service crashes with no output